### ISSUE TITLE
Fully implement parameters for user-defined functions in Calc

### ISSUE THREAD

**Issue by rlwhitcomb** [rlwhitcomb/utilities#24]
The grammar is all there in Calc.g4, and there are TODO-type comments, but the implementation is not done yet.

In addition, the library functions in "test/files" do not properly declare nor use parameters as they should (once this is finished, of course).

**Comment by rlwhitcomb** [comment]
I can't figure out how to do this, so there is a new issue for it:  [https://github.com/rlwhitcomb/utilities/issues/33](url)
Going to close this particular issue, since parameters are completely done and dusted.
